### PR TITLE
calculate works

### PR DESCRIPTION
**Pull request by EmKalin**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    
+	// TODO: Implement your solution here and return proper value
+
+	std::string result;
+    if (command == "add"){
+    	result = std::to_string(first+second);
```
Comment (WBigaj) [inline module1/homework/calculate/calculate.hpp:10]:
```suggestion
    	result = std::to_string(first + second);
```

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+    } else if (command == "subtract"){
+     	result = std::to_string(first-second);
```
Comment (WBigaj) [inline module1/homework/calculate/calculate.hpp:12]:
```suggestion
     	result = std::to_string(first - second);
```

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,20 @@
...
+    } else if (command == "multiply"){
+   	result = std::to_string(first*second);
```
Comment (WBigaj) [inline module1/homework/calculate/calculate.hpp:14]:
```suggestion
   	result = std::to_string(first * second);
```

File: module1/homework/calculate/calculate.hpp [hunk 4]
```diff
@@ -2,6 +2,20 @@
...
+    } else if (command == "divide"){
+	result = (second == 0 ? "Division by 0" : std::to_string(first/second));
```
Comment (WBigaj) [inline module1/homework/calculate/calculate.hpp:16]:
```suggestion
	result = (second == 0 ? "Division by 0" : std::to_string(first / second));
```

**Comment by ziobron** [comment]
{ "requirement_ids": [1429], "points": [5] }

+5 XP

**Pull request closed without merging**